curvefs/src/metaserver/inode_storage.cpp

```diff
@@ -158,25 +163,28 @@ MetaStatusCode InodeStorage::GetXAttr(const Key4Inode& key, XAttr *xattr) {
 MetaStatusCode InodeStorage::Delete(const Key4Inode& key) {
     WriteLockGuard writeLockGuard(rwLock_);
     std::string skey = conv_.SerializeToString(key);
-    if (!FindKey(skey)) {
-        return MetaStatusCode::NOT_FOUND;
-    }
+    // if (!FindKey(skey)) {
+    //     return MetaStatusCode::NOT_FOUND;
+    // }
 
     Status s = kvStorage_->HDel(table4inode_, skey);
     if (s.ok()) {
         EraseKey(skey);
         return MetaStatusCode::OK;
+    } else if (s.IsNotFound()) {
```

Comment:
The status of `HDel()` will never be `NOT FOUND`, it only append a DELETE record in rocksdb.

Reply:
fix